docs/usage/cli_release.md

```diff
@@ -34,6 +35,8 @@ optional arguments:
                         changes made by qgis-plugin-ci.
   -d --disable-submodule-update
                         If omitted, a git submodule is updated. If specified, git submodules will not be updated/initialized before packaging.
+  --plgin-repo-url REPO_URL
```

Comment:
```suggestion
  --plugin-repo-url REPO_URL
```

Reply:
I renamed the flag to `--alternative-repo-url`